contracts/colony/ColonyExpenditure.sol

```diff
@@ -88,21 +88,38 @@ contract ColonyExpenditure is ColonyStorage {
     public
     stoppable
     expenditureExists(_id)
-    expenditureActive(_id)
+    expenditureDraft(_id)
```

Comment:
Again, seems like this should be allowed if the expenditure is locked. If it's locked, and something changes so that you no longer want to pay it out any more... what are you meant to do?

Reply:
See above comment

Reply:
Again, you don't think the owner should be able to?

Reply:
Mm I suppose there are valid scenarios where an expenditure may need to be cancelled prior to finalization.

Here's my suggestion: introduce a compound state, `Active`, defined as either `Draft` or `Locked` and condition these functions on that.

Reply:
As per your comment May 17, the owner should not be able to cancel while locked.